assets/js/helper.js

```diff
@@ -0,0 +1,22 @@
+(function (global) {
+	'use strict';
+
+	global.getDocumentHeight = function () {
```

Comment:
Do you feel like setting up a little bundler like Browserify (or something even lighter would be better)? I’m not too concerned about JS size, and we are starting to do more and more in JS so would be nice to be able to write clean scoped JS.